educe/stac/fusion.py

```diff
@@ -98,6 +100,7 @@ def __init__(self, doc,
                                   discourse_anno.origin)
         # to be fleshed out
         self.turn = None
+        self.tstar = None
```

Comment:
I think the class docstring needs a fix ("... from the unit and discourse-level unit-level annotations...").